src/Morpho.sol

```diff
@@ -370,15 +369,15 @@ contract Morpho is IMorphoStaticTyping {
             );
 
             if (seizedAssets > 0) {
-                repaidAssets =
+                uint256 intermediateVar =
```

Comment:
Yes but wde do want to name variables even if they are intermediary
And `repaidAssets` is taken, so i suggest something more evocative than `intermediateVar`
I'm open to other suggestions but I am against having an unnamed variable floating in the code

Reply:
hmm ok I'll try to find something, but I vero repaidAssetUp

Reply:
In this specific case I don't mind the var name, since it's purely due to how solidity compiles those nested calls. 

It could be `intermediateVarToAvoidStackTooDeep`.

It could also be (note the multiplication with incentive factor has been moved down):
```solidity
uint256 seizedAssetsInSupplyAssets = seizedAssets.mulDivUp(collateralPrice, ORACLE_PRICE_SCALE);
repaidShares = seizedAssetsInSupplyAssets.wDivUp(liquidationIncentiveFactor).toSharesDown(market[id].totalBorrowAssets, market[id].totalBorrowShares);
```

Reply:
`seizedAssetsQuoted`, `seizedAssetsInLoan` is ok @adhusson 

I don't get what's wrong with `repaidAssetsUp`, which is indeed `repaidAssets` as defined below but assigned earlier to another var